cluster/addons/fluentd-gcp/fluentd-gcp-image/fluent.conf

```diff
@@ -42,11 +39,6 @@
 # the name of the Kubernetes container regardless of how many times the
 # Kubernetes pod has been restarted (resulting in a several Docker container IDs).
 
-# Do not directly collect fluentd's own logs to avoid infinite loops.
```

Comment:
I assume it hasn't worked, that's why you are removing it, am I right? Do we want to collect fluentd logs?